data-prepper-plugins/s3-source/src/main/java/com/amazon/dataprepper/plugins/source/S3ObjectReference.java

```diff
@@ -38,4 +40,29 @@ String getKey() {
     public String toString() {
         return "[bucketName=" + bucketName + ", key=" + key + "]";
     }
+
+    public Optional<String> getBucketOwner() {
```

Comment:
nit: make this package-private and keep all the getters at same place. `toString` method is in between getters.